pkgs/by-name/wa/walker/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  pkg-config,
+  gobject-introspection,
+  wrapGAppsHook,
+  gtk4,
+  gtk4-layer-shell,
+}:
+
+buildGoModule rec {
+  pname = "walker";
+  version = "0.0.68";
+
+  src = fetchFromGitHub {
+    owner = "abenz1267";
+    repo = "walker";
+    rev = "v${version}";
+    hash = "sha256-nLCFGrauMKm9NzOlzrprA8KL9CKs3nTjerEaC5992qQ=";
+  };
+
+  vendorHash = "sha256-zDntJ695k8dbwyFXbg9PapWD335MHrWbep1xxzXNIL4=";
+
+  nativeBuildInputs = [
+    pkg-config
+    gobject-introspection
+    wrapGAppsHook
```

Comment:
```suggestion
    wrapGAppsHook4
```

Reply:
Also pay attention to the problems that may be caused by using wrapper: other applications started by this launcher will also propagate environment variables. (But I still recommend that you add the wrapper first)